elixir/maybe_hug.ex

```diff
@@ -0,0 +1,8 @@
+import IO
+require Cutie
+
+cutie_person = %{:name => "Jonh",:accepts => true}
+not_cutie_person = %{:name => "Mary",:accepts => false}
+
+puts Cutie.hug(cutie_person)
+puts Cutie.hug(not_cutie_person)
```

Comment:
`s/not_cutie_person/empathy_person/` and `s/cutie_person/huggable_person/`

We make no judgment about whether they're cuties or not -- they are always cuties. The question is whether to hug them or empathise!